pkg/optimizelyjson/optimizely_json_test.go

```diff
@@ -0,0 +1,198 @@
+package optimizelyjson
+
+import (
+	"github.com/stretchr/testify/suite"
+	"testing"
+)
+
+type OptimizelyJsonTestSuite struct {
+	suite.Suite
+	jsonRepr       map[string]interface{}
+	dynamicList    []interface{}
+	optimizelyJson *OptimizelyJSON
+}
+
+func (suite *OptimizelyJsonTestSuite) SetupTest() {
+
+	suite.dynamicList = []interface{}{"1", "2", 3.01, 4.23, true}
+	suite.jsonRepr = map[string]interface{}{
+		"field1": 1,
+		"field2": 2.5,
+		"field3": "three",
+		"field4": map[string]interface{}{"inner_field1": 3, "inner_field2": suite.dynamicList},
```

Comment:
Can you please add a test case, how I can access `dynamicList`'s element. 
Ideally it should be `field4.inner_field2[2]` but I don't see `index` implementation for `GetValue` method.

Reply:
Sure, I will provide that test, although officially we won't support array object, but that does not prevent the user to include array in json .